docker/pytorch/Dockerfile

```diff
@@ -7,7 +7,7 @@ FROM twosixarmory/armory:${armory_version} AS armory-pytorch-base
 RUN /opt/conda/bin/conda install tensorflow==1.15.0 \
  pytorch==1.4 \
  torchvision==0.5.0 \
- cudatoolkit=10.1 -c pytorch
+ cudatoolkit=9.2 -c pytorch
```

Comment:
Building this locally, I still get the underlying CUDA version to be 10:
```
(base) root@453de08928f1:/usr/local/cuda# cat /usr/local/cuda/version.txt 
CUDA Version 10.0.130
```

Reply:
Check ENV for which cuda is being used in the conda environment

Reply:
Can you see what your docker container CUDA version looks like? I don't have a good sense of how to test this. Maybe we'll need to set up AWS nodes with certain binaries on them?

Reply:
```
(base) root@d20b5c67e882:/workspace# env  | grep CUDA
CUDA_PKG_VERSION=10-0=10.0.130-1
CUDA_VERSION=10.0.130
NVIDIA_REQUIRE_CUDA=cuda>=10.0 brand=tesla,driver>=384,driver<385 brand=tesla,driver>=410,driver<411
(base) root@d20b5c67e882:/workspace# env
```